src/component/page/bruker-detaljer/deltaker-detaljer/endre-deltaker-modal/AvsluttDeltakelseModal.tsx

```diff
@@ -92,6 +93,15 @@ export const AvsluttDeltakelseModal = (
         'Sluttdato er påkrevd for å sende AvsluttDeltakelse forslag'
       )
     }
+
+    const harIngenEndring = sluttDato ?
```

Comment:
Denne er litt kinkig. I teorien kunne man ha ønsket å beholde samme dato men endre på årsaken. I denne appen og bffen vet vi jo ikke hva den er, med mindre man utleder det fra tidligere endringer i historikken.

Men det er kanskje like greit å holde det enkelt slik som dette.

Reply:
ja, enten burde ikke sjekke på noe eller så blir det bare sluttdato, for å dirve å lete i historikken kunne vi ha gjort mange av modalene, men er litt for knot til at det vinner så mye